sedaro/README.md

```diff
@@ -217,14 +239,16 @@ with SedaroApiClient(api_key=API_KEY) as sedaro:
     )
 
     # create a celestial target in a branch
+    sun = {
+        'name': 'Sun',
+        'polynomialEphemerisBody': 'SUN',
```

Comment:
Please remove this line as this is now defined at the Scenario level

Reply:
Thanks! 

Just a plug for quickly coming back to removing the temporary removing extra fields. I used the same example I had in the raw request test, but because we're ignoring extra fields, nothing told me that was wrong. I updated it in the test too.